packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -339,7 +339,11 @@ namespace Tpetra {
     ///   default values.
     CrsGraph (const Teuchos::RCP<const map_type>& rowMap,
               const size_t maxNumEntriesPerRow,
-              const ProfileType pftype = DynamicProfile,
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
+	      ProfileType pftype = DynamicProfile,
```

Comment:
Could we factor out the default value and put it in a common place, ideally where the `ProfileType` enum gets defined, in `Tpetra_ConfigDefs.hpp`?  That would avoid possible typos with macros, or people adding constructors and forgetting to use the macros.  Also, now is our chance to make these `const`.

Reply:
Yep, we can do all those things.